# This service request is the position of an obstacle that should be taken a picture of.

float64 obstaclePosX
float64 obstaclePosY
---
uint8 success
